native/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -0,0 +1,151 @@
+
+            } else {
+                Project balProject = ProjectLoader.loadProject(Paths.get(this.sourcePath), projectEnvironmentBuilder);
+                String orgName = balProject.currentPackage().descriptor().org().value();
+                String projectName = balProject.currentPackage().descriptor().name().value();
+                name = orgName + "." + projectName;
+            }
+        } catch (ProjectException e) {
+            errStream.println("The current directory is not a Ballerina project!");
+            return;
+        }
+        if (!Files.exists(Paths.get(sourcePath, configPath))) {
+            try {
+                createConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when creating the Config.toml file: " + e.getMessage());
+            }
+        } else {
+            try {
+                updateConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when updating the Config.toml file: " + e.getMessage());
+            }
+        }
+    }
+    private void createConfigToml() throws Exception {
+        SyntaxTree toml = CreateSyntaxTree.createToml();
+        writeOutputFile(toml, Paths.get(this.sourcePath, "Config.toml").toAbsolutePath().toString());
+    }
+
+    private void updateConfigToml() throws Exception {
+        SyntaxTree toml = CreateSyntaxTree.updateToml(Paths.get(this.sourcePath, this.configPath), this.name);
+        writeOutputFile(toml, Paths.get(this.sourcePath, "Config.toml").toAbsolutePath().toString());
+    }
+
+    private void writeOutputFile(SyntaxTree syntaxTree, String outPath) throws Exception {
+        String content = "";
+        Path pathToFile = Paths.get(outPath);
+        Files.createDirectories(pathToFile.getParent());
+        try {
+            content = Formatter.format(syntaxTree.toSourceCode());
+        } catch (FormatterException e) {
+            throw e; //
+        }
+        try (PrintWriter writer = new PrintWriter(outPath, StandardCharsets.UTF_8.name())) {
+            writer.println(content);
+        } catch (IOException e) {
+            throw e;
+        }
+    }
+
+    public void setSourcePath(String sourceDir) {
+        this.sourcePath = sourceDir;
+    }
+
+    public void setEnvironmentBuilder(ProjectEnvironmentBuilder projectEnvironmentBuilder) {
+        this.projectEnvironmentBuilder = projectEnvironmentBuilder;
+    }
+
+    @Override
+    public void setParentCmdParser(CommandLine parentCmdParser) {
+        this.parentCmdParser = parentCmdParser;
+    }
+    @Override
+    public String getName() {
+        return PersistToolsConstants.COMPONENT_IDENTIFIER;
+    }
+    
+    @Override
+    public void printLongDesc(StringBuilder out) {
+        out.append("Generate database configurations file inside the Ballerina project")
+                .append(System.lineSeparator());
+        out.append(System.lineSeparator());
```

Comment:
Maybe this line break is not necessary?

Reply:
Updated in [e4140c3](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4140c3eaed66b42da875af1efb0cf2358ae3f01)